### DISCUSSION TITLE
command not found error

### DISCUSSION THREAD

**Discussion by SwapnilVicky** [ROM-builders/temporary#14945] [Q&A]
getting this error after fixing sync error : https://cirrus-ci.com/task/6386050976186368?logs=build#L21
my branch and commands seems fine here : https://github.com/ROM-builders/temporary/blob/miatoll-LineageOS-lineage-20.0-SwapnilVicky/build_rom.sh

Thanks in advance sir!

**Comment by Apon77** [comment]
may be again running will fix, no idea why this happened

**Reply by SwapnilVicky** [reply]
okay i will try again
